docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc

```diff
@@ -0,0 +1,49 @@
+= Activate AppCat Exoscale DBaaS On {zone}
+
+This page describes how AppCat Database as a Service on Exoscale can be enabled on a {zone}.
+
+. Set variables.
++
+[source,bash]
+----
+export CLUSTER_ID = <cluster>
+export TENANT_ID = <tenant>
+----
+
+. Login to Vault.
++
+[source,bash]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=oidc
+----
+
+. Setup API keys for exoscale cloud provider in case no other AppCat services are running in {zone}.
++
+.. Login to portal.exoscale.com
+.. Create a new organization using the naming scheme `VSHN AppCat ${CLUSTER_ID}`, for example `VSHN AppCat ch-gva-2-0`.
+.. Create an Exoscale support ticket to link the organization to the root organization.
+.. Once they're linked, in the new organization, create a restricted IAM API key using a name like `appcat-dbaas-cloudscale-ch-gva-2-0`.
+  The key should have following service permissions: `DBaaS - unrestricted`, `IAM - unrestricted`.
```

Comment:
Maybe add a note that if buckets are also needed, that it should also have bucket access?

Reply:
I added it in the warning clause, do you thing I should change the approach how I lay out the page?

Reply:
Ah right, I was looking at it in the source, so I somehow skipped that as asciidoc code :D